metadata.json

```diff
@@ -104,7 +104,9 @@
       "version_requirement": ">=4.5.0 <5.0.0"
     }
   ],
-  "dependencies": [],
+  "dependencies": [
+   {"name": "puppetlabs-app_modeling"}
```

Comment:
lol, yup. this was an attempt to make sure dsl functions were available before. It turns out to not be required